load/FilenameParser.py

```diff
@@ -488,19 +488,8 @@ def __init__(self, config):
 	def process3(self, filesets):
 		db = SQLUtility(self.config)
 		self.chapterMap = db.selectMap("SELECT id, chapters FROM books", None)
-		## I am not certain the LXX actually exists
-		extras = {"FRT":6, "INT":1, "BAK":2, "LXX":1, "CNC":2, "GLO":26, "TDX":1, "NDX":1, "OTH":5, 
-			"XXA":4, "XXB":3, "XXC":1, "XXD":1, "XXE":1, "XXF":1, "XXG":1}
-		self.chapterMap.update(extras)
-		corrections = {"MAL":3, "MAN":1, "PS2":1, "BAR":5, "EZA":9, "SIR":52}
-		self.chapterMap.update(corrections)
 		self.maxChapterMap = self.chapterMap.copy()
-		corrections = {"JOL":4, "PSA":151, "MAL":4, "BAR":6}
-		self.maxChapterMap.update(corrections)
 		self.usfx2Map = db.selectMap("SELECT id_usfx, id FROM books", None)
-		extras = {"FR":"FRT", "IN":"INT", "BK":"BAK", "CN":"CNC", "GS":"GLO", "TX":"TDX", "OH":"OTH",
-			"XA":"XXA", "XB":"XXB", "XC":"XXC", "XD":"XXD", "XE":"XXE", "XF":"XXF", "XG":"XXG"}
-		self.usfx2Map.update(extras)
 		self.usfx2Map["J1"] = "1JN" ## fix incorrect entry in books table
```

Comment:
sorry... please also remove this usfx2Map correction.

Reply:
also, for the code that is removed, please add DML that I can run on NEWDATA (DML = Data Manipulation Language.. the update commands to change the DB associated with the code we are removing)

Reply:
@bradflood I have removed this line as well and I have updated the PR description to add the DML (for our case I have created the updates statements)